src/endpoints/referral/click_count.rs

```diff
@@ -38,14 +38,14 @@ pub async fn handler(
     let mut output = Data { counts: vec![] };
     let mut i = 0;
     loop {
-        let start_naive_dt =
-            NaiveDateTime::from_timestamp_opt(query.since_day + i * query.spacing, 0).unwrap();
-        let start_time = Utc.from_utc_datetime(&start_naive_dt);
+        let _start_date_time =
```

Comment:
Don't add a "_" at the start of the function name

Reply:
okay